### PR TITLE
feat:home page features added

### PR DESCRIPTION
**Pull request by aishwarya-bm**
created home page with banner & categories list
Note: in header, login/logout will toggle, right now both are showed

https://user-images.githubusercontent.com/66695068/161124355-6b136486-a181-4537-8a9f-badc2f562432.mp4

### CODE REVIEW

File: src/components/categories/Categories.jsx [hunk 1]
```diff
@@ -34,7 +40,7 @@ export default function Categories() {
                       </Link>
 
                       <Link
-                        to={{}}
+                        to={`/explore/${categoryName}`}
```
Comment (shubhamkumar648) [inline src/components/categories/Categories.jsx:43]: i like you update route dynamically

**Review by shubhamkumar648 — APPROVED** [review]
nice ui  and good use of  grid property

File: src/pages/home/Home.jsx [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-    <div>
+    <>
       <Header />
+      <SideNav />
+      <main className="home-main">
+        <div className="home-banner p-rel">
+          <img
+            className="home-img"
+            src="https://assets.nflxext.com/ffe/siteui/vlv3/d0982892-13ac-4702-b9fa-87a410c1f2da/db219e1b-a9a8-43d7-befa-723b8a6164c1/IN-en-20220321-popsignuptwoweeks-perspective_alpha_website_large.jpg"
+          />
```
Comment (sadathshariff) [inline src/pages/home/Home.jsx:17]: Add alt description to your image

**Review by sadathshariff — APPROVED** [review]
Overall the code looks clean and good

File: src/pages/home/Home.jsx (hunk 2 above)
Comment (aishwarya-bm) [reply src/pages/home/Home.jsx:17]: thanks, added alt

**Pull request merged**